roles/ceph-common/tasks/installs/prerequisite_rhcs_cdn_install_debian.yml

```diff
@@ -13,5 +13,5 @@
   with_items:
     - { repo: "MON", configure: (mon_group_name in group_names) }
     - { repo: "OSD", configure: (osd_group_name in group_names) }
-    - { repo: "Tools", configure: (rgw_group_name in group_names or mds_group_name in group_names) }
+    - { repo: "Tools", configure: (rgw_group_name in group_names or mds_group_name in group_names or mgr_group_name in group_names) }
```

Comment:
Add `nfs_group_name in group_names or iscsi_gw_group_name in group_names or client_group_name in group_names` too.

Reply:
Also, edit https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/tasks/installs/prerequisite_rhcs_cdn_install.yml#L15 to add `mgr_group_name in group_names`. Thanks.

Reply:
@leseb sure will make changes and repush, thanks!